src/averageAges.js

```diff
@@ -55,7 +85,35 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const findRelative = function(momName, array) {
+    return array.find(el => el.name === momName);
+  };
+
```

Comment:
The same function as on 57 line - move it to global level and reuse throughout your code